gobblin-utility/src/main/java/gobblin/util/AvroUtils.java

```diff
@@ -156,4 +170,116 @@ public static GenericRecord convertRecordSchema(GenericRecord record, Schema new
       closer.close();
     }
   }
+
+  /**
+   * Get the latest avro schema for a directory
+   * @param directory the input dir that contains avro files
+   * @param conf configuration
+   * @param latest true to return latest schema, false to return oldest schema
+   * @return the latest/oldest schema in the directory
```

Comment:
The return result would be null. In method `nullifyFieldsForSchemaMerge`, if the oldSchema is null, we will retain the new schema.